autoinstall-schema.json

```diff
@@ -361,6 +361,9 @@
                 "additionalProperties": false
             }
         },
+        "drivers": {
+            "type": "boolean"
```

Comment:
I think it's not immediately obvious what `true` or `false` would mean for `drivers`. Would you consider either of these?
```json
"install_drivers" : { "type": "boolean" }
```
```json
"drivers": {
    "type": "object",
    "properties": {
        "install": {
            "type": "boolean"
        }
    }
}
```

Reply:
Yeah that would probably be better (and more future proof)